.drone.yml

```diff
@@ -3119,6 +3075,68 @@ steps:
         make change-amis-to-public-ent
         make change-amis-to-public-ent-fips
 
+  # all mandatory steps for a release promotion need to go BEFORE this step,
+  # as there is a chance that everything afterwards will be skipped
+  - name: Skip RPM/DEB publishing for alpha/beta/dev/rc builds
```

Comment:
Nit: Not sure if this name actually appears anywhere but since "skip" is just one of 2 potential outcomes, something like "Determine whether release RPM/DEB need to be published" would probably be a bit more fitting stage name.

Also, is it both rpm's and deb's like mentioned here, or just rpm's? Below it mentions rpm's only.

Reply:
Good point, changed. It's only RPMs currently, I was just getting ahead of myself. I've updated everything to only reference RPMs for now; I can update it again when we add DEBs.